src/ListView/features/CreatePage/CreatePageForm.jsx

```diff
@@ -0,0 +1,125 @@
+import React, { useEffect, useState } from "react";
+import { colorTextErrorOnLight } from "@sailthru/design-tokens";
+import {
+  FormGroup,
+  Radio,
+  RadioGroup,
+  Subtext,
+} from "@sailthru/stui-components";
+import { Fieldset, Label, Input } from "@sailthru/stui-elements";
+
+function CreatePageForm({
+  name = "",
+  onChange,
+  isDuplicateName,
+  isNameChanged,
+}) {
+  const [namingError, setNamingError] = useState(false);
+  const [optoutNamingError, setOptoutNamingError] = useState(false);
+
+  // function to check for empty, alphanumeric, and 'oc' page names
+  const handleNamingError = function () {
```

Comment:
nit-pick: I think your code explains this well enough without the comment
```suggestion
  const handleNamingError = function () {
```